### DISCUSSION TITLE
How to embedded on a web page so that on user click > data sent to a text field too

### DISCUSSION THREAD

**Discussion by Zebrafied** [Leaflet/Leaflet#9199] [Q&A] [answered]
Hello

I’m hoping to use Leaflet.js on a not-for-profit website where the map is used by user to click on locations where there are land and coastal spaces that they would like to nominate for upgrading by the local government. 

What I’d like to be able to do is that when the person clicks on the map/drops a pin is to then have that location pass over into a text field on the same page. This is as the page will collect other free text data entries by the user (a web form) and I’d like the map text location (not long lat) from the map to pass across with that data. 

Thank you.

**Comment by Falke-Design** [comment, marked as answer]
Hi, great to hear that you find Leaflet useful!

However, this issue tracker is used for reporting bugs and discussing new features. For questions on using Leaflet, please use [gis.stackexchange.com](https://gis.stackexchange.com/questions/tagged/leaflet) or [stackoverflow](https://stackoverflow.com/questions/tagged/leaflet).

Before I close this, I wil give you some hints:
* Add a `click` listener on the leaflet map-object `map.on('click', function(e){})`, where you get the latlng over `e`
* Send the latlng to a service, which converts the latlng into a text location. Something like https://nominatim.openstreetmap.org/ui/search.html
* Add the result from the service to your form

**Reply by Zebrafied** [reply]
Thank you @Falke-Design
